.github/workflows/build.yml

```diff
@@ -14,30 +14,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [8.0]
-        node: [14.x]
-        symfony: [^4.4, ^5.2]
-        sylius: [~1.11.0]
-        mysql: [5.7, 8.0]
+        php: ["8.1", "8.2", "8.3"]
+        node: ["20.x"]
```

Comment:
How about node 18?

Reply:
Sorry i pushed that before our conversation about that